src/views/studio/MyUploadsView/UploadStatusGroup/UploadStatusGroup.style.tsx

```diff
@@ -1,9 +1,9 @@
 import { css } from '@emotion/react'
```

Comment:
Get rid of `.tsx` extension, also let's rename it to `UploadStatusGroup.styles.ts`